### PR TITLE
[Workers] Update changelog through 10-28-24

### PR DESCRIPTION
**Pull request by fhanau**
### Summary

This updates the Workers Changelog based on the latest changes in the runtime.

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

**Comment by fhanau** [comment]
Includes changes by @kentonv, @jasnell, @Frederik-Baetens

File: src/content/changelogs/workers.yaml [hunk 1]
```diff
@@ -5,15 +5,30 @@ productLink: "/workers/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+  - publish_date: '2024-10-21'
+    description: |-
+      - Fixed encoding of WebSocket pong messages when talking to remote servers. Previously, when a Worker made a WebSocket connection to an external server, the server may have prematurely closed the WebSocket for failure to respond correctly to pings. Client-side connections were not affected.
```
Comment (irvinebroque) [inline src/content/changelogs/workers.yaml:10]: This was actually even more specific than this, right? It was specifically one WS server implementation that was spec compliant in a way that others weren't?

File: src/content/changelogs/workers.yaml [hunk 2]
```diff
@@ -5,15 +5,30 @@ productLink: "/workers/"
...
+  - publish_date: '2024-10-14'
+    description: |-
+      - Updated v8 to version 13.0.
   - publish_date: "2024-09-26"
     description: |-
       -  You can now connect your GitHub or GitLab repository to an existing Worker to automatically build and deploy your changes when you make a git push with [Workers Builds](/workers/ci-cd/builds/).
+  - publish_date: '2024-09-20'
+    description: |-
+      - The new `handle_cross_request_promise_resolution` compatibility flag enables the ability to resolve promises across requests without losing the correct request context.
```
Comment (irvinebroque) [inline src/content/changelogs/workers.yaml:19]: @jasnell this compat flag is not documented? https://github.com/cloudflare/workerd/pull/2719

File: src/content/changelogs/workers.yaml [hunk 3]
```diff
@@ -5,15 +5,30 @@ productLink: "/workers/"
...
       -  Revamped Workers and Pages UI settings to simplify the creation and management of project configurations. For bugs and general feedback, please submit this [form](https://forms.gle/XXqhRGbZmuzninuN9).
   - publish_date: '2024-09-16'
     description: |-
       - Updated v8 to version 12.9.
+  - publish_date: '2024-08-27'
+    description: |-
+      - Introduced a new overloaded error message for Durable Objects:
+      "Durable Object is overloaded. Too many requests for the same object within a 10 second window."
+      This error message does not replace other types of overload messages that you may encounter for
+      your Durable Object, and is only returned at more extreme levels of overload.
```
Comment (irvinebroque) [inline src/content/changelogs/workers.yaml]: I think this is covered here?

https://developers.cloudflare.com/durable-objects/platform/changelog/#2024-09-07

@vy-ton

**Review by irvinebroque — APPROVED** [review]

File: src/content/changelogs/workers.yaml (hunk 1 above)
Comment (fhanau) [reply src/content/changelogs/workers.yaml:10]: Deferring to @kentonv who authored the change and release note.

File: src/content/changelogs/workers.yaml (hunk 3 above)
Comment (fhanau) [reply src/content/changelogs/workers.yaml]: You're right, wasn't aware of the DO changelog. Dropping it here then.

**Pull request merged**